x-pack/plugins/lens/public/indexpattern_plugin/indexpattern.tsx

```diff
@@ -13,6 +13,7 @@ import { Datasource, DataType } from '..';
 import uuid from 'uuid';
 import { DatasourceDimensionPanelProps, DatasourceDataPanelProps } from '../types';
 import { getIndexPatterns } from './loader';
+import { ChildDragDropProvider, DragDrop, DragContextState } from '../drag_drop';
```

Comment:
I would recommend creating a mock of the `drag_drop` library so that you can write a test like:

```typescript
import { DragDrop } from '../drag_drop';

jest.mock('../drag_drop');

it('should call setState on drop', () => {
  wrapper.find(DragDrop).prop('onDrop')!({
    operationId: 'droppedOperation',
  });

  expect(setState).toHaveBeenCalledWith({
    ...state,
    columns: {
      ...state.columns,
      newColumnId: expect.objectContaining({
        operationId: 'droppedOperation',
      }),
    }
  });
});
```